### ISSUE TITLE
Split the glossary between Pharo et OO

### ISSUE THREAD

**Issue by jecisc** [pharo-open-documentation/pharo-wiki#188]

**Comment by juliendelplanque** [comment]
You mean 2 distinct pages?

**Comment by jecisc** [comment]
Maybe not 2 pages but at least two different sections

**Comment by juliendelplanque** [comment]
Ok, I agree and know which tag to put on this issue now 💯

**Comment by jecisc** [comment]
@juliendelplanque I tried to fix this issue but I'm not sure it's a good thing anymore.

While checking what should go in which category I found out that those two categories are maybe not good. 

First, because some things are part of the object-oriented model, but does not apply to every OO model. For example: Messages. It's part of the OO model but other languages do not use message sent. 
Second, because some things are not part of the OO concept but are not necessarily tight to Pharo. For example the concept of virtual machine. 

Maybe it would be better to not do categories?
